src/schemas/Config.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+/**
+ * Connector configuration schema.
+ */
+export class Config {
+  /**
+   * ILP address of the connector. This property can be omitted if an account with `relation=parent` is configured under `accounts`.
+   */
+  ilpAddress?: string;
+  /**
+   * Description of connector account plugins.
+   */
+  accounts: {
+    /**
+     * Description of individual account.
+     */
+    [k: string]: {
+      /**
+       * Relationship between the connector and the counterparty that the account is with.
+       */
+      relation: "parent" | "peer" | "child";
+      /**
+       * Name of the ILP plugin that should be used for this account.
+       */
+      plugin: string;
+      /**
+       * Currency code or other asset identifier that will be passed to the backend to select the correct rate for this account.
+       */
+      assetCode: string;
+      /**
+       * Interledger amounts are integers, but most currencies are typically represented as fractional units, e.g. cents. This property defines how many Interledger units make up one regular units. For dollars, this would usually be set to 9, so that Interledger amounts are expressed in nanodollars.
+       */
+      assetScale: number;
+      /**
+       * Options that will be passed to the plugin constructor.
+       */
+      options?: {
+        [k: string]: any;
+      };
+      /**
+       * What segment will be appended to the connector's ILP address to form this account's ILP address. Only applicable to accounts with `relation=child`. Defaults to the id of the account, i.e. the key used in the `accounts` config object.
+       */
+      ilpAddressSegment?: string;
+      [k: string]: any;
+    };
+  };
+  /**
+   * Which accounts should be use as peers for routing purposes. Defaults to the set of accounts set to `relation=peer`.
+   */
+  peers?: string[];
+  /**
+   * Additional routes to add to the connector's routing table.
+   */
+  routes?: {
+    /**
+     * ILP address prefix that this route applies to. Configured routes take precedence over the same or shorter prefixes that are local or published by peers. More specific prefixes will still take precedence.
```

Comment:
The fact that more specific prefixes always take precedence seems potentially problematic. If I want to configure my connector to always send `g.crypto.xrp.` to you, I don't want a route broadcast to be able to override that just by including a long list of specific XRP addresses.

Reply:
Yes, this is a problem with the current routing protocol and will be fixed when we introduce authenticated routes.